### PR TITLE
Add Digital Science to adopters

### PR DESCRIPTION
**Pull request by sdlarsen**
Signed-off-by: Søren Dalby Larsen <soren@uberresearch.com>

#### What type of PR is this?

/kind other

#### What this PR does / why we need it:

Adding Digital Science to adopters.

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by haircommander** [comment]
/ok-to-test

Thanks @sdlarsen, LGTM !

@TomSweeneyRedHat PTAL

File: ADOPTERS.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ Below is a non-exhaustive list of CRI-O adopters in production environments:
 * [openSUSE Kubic](https://kubic.opensuse.org) is a Certified Kubernetes
   distribution based on openSUSE MicroOS. It uses CRI-O as a supported container
   runtime for Kubernetes.
+* [Digital Science](https://www.digital-science.com/) is using CRI-O as the runtime in data processing clusters behind [Dimensions](https://www.dimension.ai) due to it being just enough runtime for kubernetes, and the flexibility to use more than runc.
```
Comment (haircommander) [inline ADOPTERS.md:14]: ooh actually, one nit: I believe Kubernetes should be capitalized

**Comment by haircommander** [comment]
/approve

LGTM

**Review by giuseppe — APPROVED** [review]
/lgtm

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**